boltdesignsystem/bolt
# DS 601 shorthand template prune

## Jira

https://pegadigitalit.atlassian.net/browse/DS-601

## Summary

The `'@bolt/[COMPONENT-NAME].twig` shorthand pattern was switched to The `'@bolt-components-[COMPONENT-NAME]/[COMPONENT-NAME].twig'`

## Details

Below examples were switched

- @bolt/button-group.twig
- @bolt/button.twig
- @bolt/card.twig
- @bolt/card-w-teaser.twig
- @bolt/eyebrow.twig
- **_@bolt/flag.twig_** &#x1F538;
- @bolt/headline.twig
- @bolt/image.twig
- @bolt/link.twig
- @bolt/teaser.twig
- @bolt/text.twig
- @bolt/video.twig

Then the codebase was searched for more shorthand examples.

&#x1F538; **_@bolt/flag.twig_** - the flag is an object, it’s a legacy thing we didn’t deprecate yet. Places with this shorthand usage have been marked in a code.

## How to test

Pull the branch make sure there are no more shorthand pattern examples. Check if there aren't any regressions.

## Release notes

The `'@bolt/[COMPONENT-NAME].twig` shorthand pattern won't be longer in use. A longhand pattern is required to use twig's includes  (`'@bolt-components-[COMPONENT-NAME]/[COMPONENT-NAME].twig'`)
